shapes/contacts/example/Group/Group_1.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix :       <#>.
+@prefix vcard:  <http://www.w3.org/2006/vcard/ns#>.
+
+:this
+    a vcard:Group;
+    vcard:fn "Group 1";
+    vcard:hasMember <../Person/67fca8b3-c74a-499e-b00f-67c543fa533f/index.ttl#this>.
```

Comment:
Didn't you have a change that points to another example file that exists?

Reply:
Yeah, I've replaced `/example` now with what was in `/example2`, so this problem should be gone.